modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -219,26 +206,14 @@ public void onReceived(NetworkMessage message, NetworkAddress senderAddr, @Nulla
                         if (ex == null) {
                             msg = REPLICA_MESSAGES_FACTORY.replicaResponse().result(res).build();
                         } else {
-                            var traceId = UUID.randomUUID();
-
-                            LOG.warn("Exception was thrown [traceId={}]", ex, traceId);
+                            LOG.warn("Failed to process replica request [request={}, err={}]", request, ex);
```

Comment:
In that way, we won't see a stack in the node where an exception happens.
It is suspecting approach: to see a stack in that node where the exception is not thrown.

Reply:
Nice catch! Fixed!